nncf/experimental/tensor/functions/torch_numeric.py

```diff
@@ -299,3 +330,41 @@ def _(a: torch.Tensor, axis: int = -1, descending=False, stable=False) -> torch.
 @numeric.diag.register(torch.Tensor)
 def _(a: torch.Tensor, k: int = 0) -> torch.Tensor:
     return torch.diag(a, diagonal=k)
+
+
+@numeric.logical_or.register(torch.Tensor)
+def _(x1: torch.Tensor, x2: torch.Tensor) -> torch.Tensor:
+    return torch.logical_or(x1, x2)
+
+
+@numeric.zero_elements.register(torch.Tensor)
+def zero_elements(x: torch.Tensor) -> torch.Tensor:
+    return torch.abs(x) < torch.finfo(x.dtype).eps
+
+
+@numeric.masked_mean.register(torch.Tensor)
+def _(
+    x: torch.Tensor, mask: Optional[torch.Tensor], axis: Union[int, Tuple[int, ...], List[int]], keepdims=False
+) -> torch.Tensor:
+    if mask is None:
+        return torch.mean(x, axis=axis, keepdims=keepdims)
+    device = x.device
+    masked_x = np.ma.array(x.detach().cpu().numpy(), mask=mask.detach().cpu().numpy())
```

Comment:
It looks like not optimal implementation for torch backend in practice for gpu. Could you think about more optimal implementation without copying tensor in gpu case?

Reply:
@daniil-lyakhov What limitations in torch led to the need to use numpy?

Reply:
As `torch.masked_tensor` is an prototype (https://pytorch.org/docs/stable/masked.html) and does not implement neither median nor quantile, I decided to use numpy instead. As for now, it seems possible to use `torch.mean` with a torch masked tensor

Reply:
https://github.com/openvinotoolkit/nncf/blob/24f0ee707cd5a35abf351f8647efc0fbd0ee273b/nncf/torch/tensor_statistics/collectors.py#L90-L100

Reply:
I'm not sure, but it looks like you can implement it something like that:
```
num = torch.sum(mask, axis)
y = torch.sum(x * mask, axis) / num
```

Reply:
reworked by torch.nanmean